zowe-cli-id-federation-plugin/src/api/templates/tss_refresh.jcl

```diff
@@ -0,0 +1 @@
+TSS REFRESH
```

Comment:
I suppose we should do nothing or `F TSS,REFRESH(IDMAPTAB)`

https://techdocs.broadcom.com/us/en/ca-mainframe-software/security/ca-top-secret-for-z-os/16-0/administrating/issuing-commands-to-communicate-administrative-requirements/keywords/idmap-keyword-implement-z-os-identity-propagation-mapping.html

Not sure what is better ...